src/utils/formValidations.js

```diff
@@ -135,11 +162,43 @@ export const ValidatePasswordValue = {
     checkSpecialNumber: (password) => {
       const isInvalid = /[!@#$%^&*)(+=._-]+/g.test(password);
 
-      if (isInvalid) {
+      if (!isInvalid) {
         return PASSWORD_FIELD_VALIDATION.REQUIRE_SPECIAL_CHARACTER;
       }
 
       return null;
     },
   },
 };
+
+export const ValidateConfirmPasswordValue = {
+  required: {
+    value: true,
+    message: PASSWORD_FIELD_VALIDATION.REQUIRED,
+  },
+  validate: {
+    checkIsSame: (confirmPassword, formValues) => {
```

Comment:
2nd parameter is wrong. you won't need whole-form values. just send the password field.